www/include/monitoring/status/ServicesHostGroups/serviceGridByHG.php

```diff
@@ -39,111 +39,115 @@
 if (!isset($oreon)) {
     exit();
 }
+include("./include/common/autoNumLimit.php");
+
+!isset($_GET["sort_types"]) ? $sort_types = 0 : $sort_types = $_GET["sort_types"];
```

Comment:
What do you write this format? 

Because me, I writting this :
`$sort_types` = !isset($_GET["sort_types"]) ? 0 : $_GET["sort_types"];`